src/App.tsx

```diff
@@ -1,167 +1,36 @@
-                </tr>
-
-                <tr data-cy="person" className="has-background-warning">
-                  <td>
-                    <a href="#/people/jan-frans-van-brussel-1761">
-                      Jan Frans van Brussel
-                    </a>
-                  </td>
-
-                  <td>m</td>
-                  <td>1761</td>
-                  <td>1833</td>
-                  <td>-</td>
-
-                  <td>
-                    <a href="#/people/jacobus-bernardus-van-brussel-1736">
-                      Jacobus Bernardus van Brussel
-                    </a>
-                  </td>
-                </tr>
-
-                <tr data-cy="person">
-                  <td>
-                    <a
-                      className="has-text-danger"
-                      href="#/people/lievijne-jans-1542"
-                    >
-                      Lievijne Jans
-                    </a>
-                  </td>
-
-                  <td>f</td>
-                  <td>1542</td>
-                  <td>1582</td>
-                  <td>-</td>
-                  <td>-</td>
-                </tr>
-
-                <tr data-cy="person">
-                  <td>
-                    <a href="#/people/bernardus-de-causmaecker-1721">
-                      Bernardus de Causmaecker
-                    </a>
-                  </td>
-
-                  <td>m</td>
-                  <td>1721</td>
-                  <td>1789</td>
-
-                  <td>
-                    <a
-                      className="has-text-danger"
-                      href="#/people/livina-haverbeke-1692"
-                    >
-                      Livina Haverbeke
-                    </a>
-                  </td>
-
-                  <td>
-                    <a href="#/people/lieven-de-causmaecker-1696">
-                      Lieven de Causmaecker
-                    </a>
-                  </td>
-                </tr>
-              </tbody>
-            </table>
+import { Navigation } from './components/Navigation';
+import { PeoplePage } from './components/PeoplePage';
+import { HomePage } from './components/HomePage';
+
+export const App = () => {
+  return (
+    <div data-cy="app">
+      <Navigation />
+      <main className="section">
+        <div className="container">
+          <div className="block">
+            <div className="box table-container">
+              <Routes>
+                <Route path="/home" element={<Navigate to="/" replace />} />
+                <Route path="/" element={<HomePage />} />
+                <Route
+                  path="*"
+                  element={
+                    <div className="container">
+                      <h1 className="title">Page not found</h1>
+                    </div>
+                  }
+                />
+                <Route path="/people" element={<PeoplePage />}>
+                  <Route path=":personHref" element={<PeoplePage />} />
+                </Route>
```

Comment:
The nested route for `PeoplePage` might not work as intended. The parent route `/people` already renders `PeoplePage`, and the nested route `:personHref` also renders `PeoplePage`. This could lead to rendering issues or unexpected behavior. Consider reviewing the logic for nested routes and ensure that the nested route is necessary and correctly implemented.